### PR TITLE
Feature/watchlist

### PR DESCRIPTION
**Pull request by scottusrobus**
Add Documentation for both our Watching and Watchlist Customer endpoints.

PHP Watchlist resource is already merged. PR for Watching:
https://github.com/vhx/vhx-php/pull/18

Node PR for both Watching and Watchlist:
https://github.com/vhx/vhx-node/pull/12

Todo
- [x]  Merge above PRs
- [x]  Add Internal Doc notes for the /watching/me and /watchlist/me aliases (when logged in via VHX auth system)

### CODE REVIEW

File: source/includes/_resource.customers.md [hunk 1]
```diff
@@ -747,3 +750,389 @@ vhx.customers.removeProduct('https://api.vhx.tv/customers/1', {
...
+<?php$watching = \VHX\Watching::items("https://api.vhx.tv/customers/1");
+```
+
+> Example Response
+
+```json
+{
+  "_links": {
+    "self": {
+      "href": "http://api.crystal.dev/customers/1/watching?page=1&per_page=50"
```
Comment (ksheurs) [inline source/includes/_resource.customers.md]: FYI some references to `api.crystal.dev` instead of `api.vhx.tv`.

**Comment by ksheurs** [comment]
Nice @scottusrobus! Just left 1 note re: API hosts.

I'm going to add `_links.watchlist.href` and `_links.watching.href` to the customer API response, so we can additionally document these new resources under "Retrieve a customer". https://app.asana.com/0/12937965751953/219241032571041

**Comment by scottusrobus** [comment]
@ksheurs thanks for reviewing!

sounds good. let me know when those are out, and I'll merge this in.

**Comment by ksheurs** [comment]
https://github.com/vhx/crystal/pull/2408

**Pull request merged**